app/code/Magento/InventorySales/Plugin/Store/WebsiteResourcePlugin.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySales\Plugin\Store;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\InventorySalesApi\Api\Data\SalesChannelInterface;
+
+class WebsiteResourcePlugin
+{
+    /**
+     * @var ResourceConnection
+     */
+    private $resource;
+
+    /**
+     * WebsiteResourcePlugin constructor.
+     * @param ResourceConnection $resource
+     */
+    public function __construct(
+        ResourceConnection $resource
+    ) {
+        $this->resource = $resource;
+    }
+
+    /**
+     * Get code from database
+     * @param \Magento\Store\Model\ResourceModel\Website $resourceModel
+     * @param int $websiteId
+     * @return string
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    private function getCodeFromDatabase(
+        \Magento\Store\Model\ResourceModel\Website $resourceModel,
+        int $websiteId
+    ) {
+        $connection = $resourceModel->getConnection();
+        $qry = $connection->select()->from($resourceModel->getMainTable(), 'code')->where('website_id = ?', $websiteId);
+        return (string) $connection->fetchOne($qry);
+    }
+
+    /**
+     * @param string $oldCode
+     * @param string $newCode
+     */
+    private function updateSalesChannel(
+        string $oldCode,
+        string $newCode
+    ) {
+        $connection = $this->resource->getConnection();
+        $tableName = $this->resource->getTableName('inventory_stock_sales_channel');
+        $connection->update($tableName, [
+            SalesChannelInterface::CODE => $newCode,
+        ], $connection->quoteInto(SalesChannelInterface::CODE . "=? and type='website'", $oldCode));
```

Comment:
1) Use `\Magento\InventorySalesApi\Api\Data\SalesChannelInterface::TYPE_WEBSITE
2) Second parameter can be array with clauses instead of `sql string`